### ISSUE TITLE
Basic info we want to collect about projects

### ISSUE THREAD

**Issue by tricky-coyote** [ProgressiveCoders/functions#20]
Several members have expressed the need for a spot where members can get a general overview of current (and perhaps past) projects that are part of ProgCode. This git issue exists for the purpose of deciding what kind of basic info should be contained in such an overview.

I'm working on a forked version of ProgCode's "master" Project database to create this spot. (The main thing I'm leaving behind with the new fork is sensitive user info.)

Here's what it looks like now: https://airtable.com/invite/l?inviteId=invyHESwh1q7Q3WQz&inviteToken=084014a222bfbc663161dff182f4cffa

Purposes so far identified for this database:

- enable would-be volunteers to see what Projects are active, what they're needing, how they run, what they're about, etc
- enable Project leads and participants to see what **features** other Projects are working on or imagining, for the ultimate aims of collaboration, integration and not reinventing 
- enable would-be app creators to see whether their ideas are already being worked on
- enable app ideas, whether or not they have a steward, to live somewhere waiting for a person or team to take interest

**How you can participate here**

- suggest additional purposes for the database
- suggest key pieces of info we want to collect about each app
- comment on any of the above
- cheerlead!

**Kicking off the list of key pieces of info we might want to collect (in addition to what's already [in the DB](https://airtable.com/invite/l?inviteId=invyHESwh1q7Q3WQz&inviteToken=084014a222bfbc663161dff182f4cffa) now)**:

- Values screen. Will you make all versions of your app available to everyone, or only to people whose politics you like? [Dropdown: Open to everyone; Only progressives; Only Democrats; Basic version open, supported version only for progressives; Other]
- Legal structure [C Corp, B Corp, LLC, Sole proprietorship, 501c3, 501c4, 527, None, Other]
- Org structure [Traditional, Cooperative, Sociocratic, Other]
- Current number of active contributors 
- Date this org's record was last updated
- Full release features. concise descriptions of each feature anticipated. Ex: mobile phonebanking, crowdsourced campaign strategy
- MVP features. concise descriptions of essential app features. Ex: Facebook login, find friends of friends
- Tools used for project management
- Mission accomplished. What success looks like.

**Comment by sreynen** [comment]
I suggest adding what license the code has. In many cases that's more important than the legal structure of the organization maintaining the project, as the code can outlive the organization.

**Comment by mattmallon** [comment]
Some suggestions off the top of my head:

- Links to project management
- Links to documentation and other resources (meeting notes, etc.)
- Technology stack (languages, frameworks, tools)
- Link to repository (if applicable)
- Link to project in its current state (if applicable)
- Meeting schedule or general approach to meetings

**Comment by dave-mahler** [comment]
One essential element of this is also that it stays updated. That means we also need to establish who, how, how often, keeps it updated. We've discussed Partner Outreach before, but keeping this update seems like a natural piece of their ongoing work.

Also, how would this link to the new project board?

**Comment by dave-mahler** [comment]
Anyone know how to add a member? Want to add @rthbound. His email is tad.hosford@gmail.com

**Comment by danodemo** [comment]
I really like the idea of categorizing projects in a way similar to how Dave categorized the channels list.

**Comment by dave-mahler** [comment]
Yeah, it's a great idea. I was organizing them by status, but a category or tagging system would also be great. As rthbound mentioned, we could use the same categories that [bernkit](http://www.bernkit.com/) used.

**Comment by danodemo** [comment]
Is that something you could start on for the weekend/moving into next week?  Do a first pass of categorization and then we'll crowdsource the staff and see if there's anything we should add, break apart, edit, etc?

**Comment by danodemo** [comment]
Or should we try and find someone to take the challenge?

**Comment by dave-mahler** [comment]
We don't need the categorizations decided to start bringing staff on board. I actually want to show everything to Pamela and Stephen soon. Use their experience to guide revisions

**Comment by dave-mahler** [comment]
I would use Bernkit categories for the first draft

**Comment by dave-mahler** [comment]
@danodemo I recreated the color scheme you made for the tech stack. Could you take a spin through it and make sure I got it right? I want to copy that column and add a column for "Tech Current Needs" to highlight which languages an app is actively looking for help on, and it doesn't make sense to copy the column until the color scheme is right. Also, can you add a description to the column that lays out the color scheme?

**Comment by JakeStevens** [comment]
+1 to sreynen's comment. Not only may the code outlive the organization, but some people may be looking specifically to contribute to open source projects.

**Comment by dave-mahler** [comment]
@JakeStevens we have a column in the sheet to collect that info. Agreed it's a great idea.

**Comment by danodemo** [comment]
**## STARTING TODAY, PLEASE DO NOT MAKE ANY CHANGES TO COLUMNS IN THE AIRTABLE**

We have started hooking up the API to our new website, so at this point whatever is there needs to stay there with nothing new added or altered.  If y'all need to change anything, PLEASE CHECK WITH ME FIRST!!!!  Even the slightest change could break the code.

The values can still be edited and changed, but the columns cannot.  Thank you!
